app/views/custom/proposals/map.html.erb

```diff
@@ -0,0 +1,52 @@
+<div class="row">
```

Comment:
I'm not sure about the benefits of copying this file; I'd rather use the original one.

If in the end we do copy it, it would be great to have one commit to copy the file and another one with the changes, so it's easier to review.